Week3/AmirHossein/homework/js-exercises/promiseMe.js

```diff
@@ -0,0 +1,58 @@
+function getData(url) {
+    fetch(url)
+      .then(response => response.json)
+      .then(json => console.log(json))
+      .catch(error => console.log(error));
+  }
+
+  getData('https://randomfox.ca/floof/');
+
+
+
+async function getData(url) {
+    try {
+        const response = await fetch(url);
+        return console.log(response)
```

Comment:
This is wrong, you're returning console.log() output which is empty. You should return the promise in getData.